src/routers/controllers/resendEmailController.ts

```diff
@@ -12,7 +12,7 @@ export const resendEmailController = async (req: Request, res: Response): Promis
     const validEmail = validateEmailString(email);
     let emailOnList = false;
     if (validEmail) {
-        emailOnList = await isEmailAuthorised(email, companyNumber);
+        emailOnList = await isEmailAuthorised(req, email, companyNumber);
```

Comment:
how are you checking this, if email is in the list as the data returned by end point is paginated. If more then 15 we would not be able to check the data. hence i have created a new filter on /associations/compamy_number endpoint/

Reply:
Yes, this function uses /associations/companies/{company_number}?user_email= endpoint